### ISSUE TITLE
Helm Releases tab empty

### ISSUE THREAD

**Issue by red8888** [vscode-kubernetes-tools/vscode-kubernetes-tools#1072]
Other tabs like Namespaces and Workloads work but Helm Releases shows nothing

<img width="154" alt="image" src="https://user-images.githubusercontent.com/6265399/167900936-350bb7a4-2e5c-40bc-ae12-81ac9053fe6c.png">

I of course confirmed `helm ls -A` showed all my releases correctly

Why is it blank in the UI? Does the UI only support a single namespace or something because I dont think I have any releases deployed to the default namespace

**Comment by lstocchi** [comment]
Hi @red8888 , yes it takes the current namespace and shows all releases belonging to it -> you can find the code here https://github.com/vscode-kubernetes-tools/vscode-kubernetes-tools/blob/92582a57ce722e166d25482b443c37a58d16305a/src/helm.exec.ts#L740

**Comment by red8888** [comment]
oh man does it support all namespaces for helm? otherwise I dont think the current implementation is super useful

**Comment by red8888** [comment]
An example implementation would be Len which supports this: https://k8slens.dev/

**Comment by bacongobbler** [comment]
`helm list --all-namespaces`

**Comment by bacongobbler** [comment]
I don't think so, no... I can only comment from the Helm side of things. I'm not too familiar with the internals of the Kubernetes extension.

**Comment by itowlson** [comment]
@lstocchi The whole tree is filtered by current namespace - would it not be weird for Helm releases to be different?
